### ISSUE TITLE
README.md refers to Escoria as if this is that project.

### ISSUE THREAD

**Issue by Sslaxx** [deep-entertainment/safehouse-game#1]
As per the subject line, really. My understanding is that this is a game using Escoria, and not Escoria itself, is this right?

**Comment by dploeger** [comment]
Yes, absolutely. Thing is, this repository is currently based on the Escoria main branch and heavily in the making (as well as Escoria itself). When godotengine/escoria#321 is done, I will rewrite the game based on the game template. 

Sorry if this causes confusion. I'll keep this issue open so this is transparent to others.
